src/script/printer/doc_printer/python_doc_printer.cc

```diff
@@ -294,7 +295,11 @@ void PythonDocPrinter::PrintTypedDoc(const LiteralDoc& doc) {
   } else if (const auto* float_imm = value.as<FloatImmNode>()) {
     // TODO(yelite): Make float number printing roundtrippable
```

Comment:
Do you know if this TODO is resolved?

Reply:
Not yet but indeed there is a way forward